.github/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,76 @@
+# Contributor Covenant Code of Conduct
```

Comment:
Why these files are under .github and not at the top level?

Reply:
it's a convention folder. Also the main idea of these files is github shows them when you create issue or PR in the side panel, I think people rarely read it directly from these files. And we also reference them in Readme but I can move to root for visibility. rest of TEMPLATE.md files will remain in .github folder right?

Reply:
done